### DISCUSSION TITLE
Is pull replication from couch 1.x not supported?

### DISCUSSION THREAD

**Discussion by ronag** [apache/couchdb#3464] [Q&A] [answered]
I'm getting timeout errors in Couch 3.1 on _revs_diff

<img width="395" alt="Screen Shot 2021-03-26 at 12 04 32" src="https://user-images.githubusercontent.com/3065230/112622612-91747b00-8e2b-11eb-96a4-788505d4eeeb.png">

**Comment by nickva** [comment, marked as answer]
Replication from 1.x is supported. I verified it with a 1.7.1 docker image replicating to latest 3.x branch.

```
docker run --name couchdb17 -p 5985:5984  couchdb:1.7.1
```

Then created a db there and could replicate from it. 

```
$ http put http://adm:pass@localhost:15984
/_replicator/from17 source=http://localhost:5985/db17 target=http://adm:pass@localhost:15984/db17_replicated create_target:='true' continuous:='true'
```

**Reply by nickva** [reply]
Try increasing `[replicator] connection_timeout = 120000` a and maybe reduce the number of workers and the batch size 
https://docs.couchdb.org/en/stable/config/replicator.html#replicator/worker_batch_size.

**Reply by ronag** [reply]
Thanks! That helped.
